CMakeLists.txt

```diff
@@ -254,8 +254,7 @@ if (OPM_ENABLE_PYTHON)
   endif()
   find_package(pybind11 2.2 CONFIG)
   if (NOT pybind11_FOUND)
-    # Use full path for reuse with pypi
-    set(pybind11_INCLUDE_DIRS ${PROJECT_SOURCE_DIR}/python/pybind11/include ${PYTHON_INCLUDE_DIRS})
```

Comment:
Might this be needed to support embedded python?

Reply:
Seems like python/pybind11/CMakeLists.txt only sets the upper case version PYBIND11_INCLUDE_DIRS.

Reply:
Indeed with this 
```
diff --git a/CMakeLists.txt b/CMakeLists.txt
index c4efb10..897655d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -255,6 +255,9 @@ if (OPM_ENABLE_PYTHON)
   find_package(pybind11 2.2 CONFIG)
   if (NOT pybind11_FOUND)
       add_subdirectory(python/pybind11)
+      if(NOT pybind11_INCLUDE_DIRS)
+        set(pybind11_INCLUDE_DIRS ${PYBIND11_INCLUDE_DIR} ${PYTHON_INCLUDE_DIRS})
+      endif()
   endif()
 endif()
 
```
it works and even fixes an issue with python on redhat for me.

Reply:
added.